### PR TITLE
ACC-285 Chore: Basic test coverage

### PR DESCRIPTION
**Pull request by thurstontye**
Applies basic test coverage. Replaces https://github.com/Financial-Times/session-decoder-js/pull/19

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,8 +17,10 @@
   "devDependencies": {
     "@financial-times/n-gage": "^3.6.0",
     "chai": "^3.5.0",
+    "lodash": "^4.17.15",
```
Comment (umbobabo) [inline package.json:20]: This one doesn't seem related.

Comment (thurstontye) [reply package.json:20]: @umbobabo I konw. The previous PR was failing test in CI because of a call to `lodash` by 'msgpack-javascript'. Worked locally but it was resolving to lodash elsewhere. Not sure what was going on. However including 'lodash' as a direct dependency seemed to fix the issue. FYI seems this is using a very old version of 'msgpack-javascript so did not want to get into the weeds in updating dependencies on this as it's not a priority.

Comment (umbobabo) [reply package.json:20]: Ok no problem 👍
